bot/exts/filters/filtering.py

```diff
@@ -2,7 +2,7 @@
 import re
 import unicodedata
 import urllib.parse
-from datetime import timedelta
+from datetime import datetime, timedelta
```

Comment:
```suggestion
from datetime import timedelta
```
If using arrow as per other suggestion then this import would not be needed :)

Reply:
https://github.com/python-discord/bot/pull/2250/commits/4eaecb32830aa122cd53656de562caf20b79e33d